### ISSUE TITLE
Negative running timers

### ISSUE THREAD

**Issue by thefosk** [openresty/lua-nginx-module#837]
What does it mean when [`ngx.timer.running_count()`](https://github.com/openresty/lua-nginx-module#ngxtimerrunning_count) returns a negative integer?

OpenResty 1.9.7.5

**Comment by agentzh** [comment]
@thefosk It means there is a bug somewhere that makes the counter go out of sync. Is it possible for you to reproduce it reliably? Or better, will you provide a minimal and standalone test case that can reproduce this issue on, say, my side?

**Comment by agentzh** [comment]
@thefosk Also, can you reproduce it with the latest OpenResty 1.9.15.1?

**Comment by thefosk** [comment]
@agentzh yes I can reproduce `1.9.15.1` and so far I can reproduce it in a `centos:7` docker container. I will try to create a minimal reproducible test case.
